### PR TITLE
UnHardcoded Discord

### PR DESCRIPTION
**Pull request by Zert0X**
## About The Pull Request

UnHardcoded Discord link for BYOND hub
Why didn't we take it from the config before?

## Why It's Good For The Game

No needs to edit code every time it's changing.

## Changelog
:cl:
config: Discord link for BYOND hub now takes from config
/:cl:

### CODE REVIEW

File: code/game/world.dm [hunk 1]
```diff
@@ -328,7 +328,8 @@ GLOBAL_VAR(restart_counter)
 		hostedby = CONFIG_GET(string/hostedby)
 
 	s += "<b>[station_name()]</b>";
-	s += "(<a href='https://discord.gg/z9ttAvA'>Discord</a>|<a href='http://beestation13.com'>Website</a>)"
+	var/discordurl = CONFIG_GET(string/discordurl)
+	s += "(<a href='[discordurl]'>Discord</a>|<a href='http://beestation13.com'>Website</a>))"
```
Comment (PowerfulBacon) [inline code/game/world.dm:332]: can the website be unhardcoded too?

Comment (Zert0X) [reply code/game/world.dm:332]: If you want to edit config, then yes

Comment (Zert0X) [reply code/game/world.dm:332]: We need to add new variable to config.
And then just take it out

Comment (Zert0X) [reply code/game/world.dm:332]: Or we can take link to the rules/forum

**Pull request merged**